core/templates/dev/head/library/library.html

```diff
@@ -80,18 +80,30 @@ <h2 class="oppia-library-group-header">
                    ng-swipe-left="scroll($index, false)">
                 <div class="oppia-library-carousel-overlay-left"></div>
                 <div class="oppia-library-carousel-tiles">
-                  <exploration-summary-tile ng-repeat="tile in group.activity_summary_dicts"
-                                            exploration-id="tile.id"
-                                            exploration-title="tile.title"
-                                            last-updated-msec="tile.last_updated_msec"
-                                            objective="tile.objective"
-                                            category="tile.category"
-                                            ratings="tile.ratings"
-                                            num-views="tile.num_views"
-                                            thumbnail-icon-url="tile.thumbnail_icon_url"
-                                            thumbnail-bg-color="tile.thumbnail_bg_color"
-                                            class="protractor-test-exp-summary-tile">
-                  </exploration-summary-tile>
+                  <div ng-repeat="tile in group.activity_summary_dicts">
+                    <exploration-summary-tile ng-if="tile.activity_type == 'exploration'"
+                                              exploration-id="tile.id"
+                                              exploration-title="tile.title"
+                                              last-updated-msec="tile.last_updated_msec"
+                                              objective="tile.objective"
+                                              category="tile.category"
+                                              ratings="tile.ratings"
+                                              num-views="tile.num_views"
+                                              thumbnail-icon-url="tile.thumbnail_icon_url"
+                                              thumbnail-bg-color="tile.thumbnail_bg_color"
+                                              class="protractor-test-exp-summary-tile">
+                    </exploration-summary-tile>
+                    <collection-summary-tile ng-if="tile.activity_type == 'collection'"
```

Comment:
You had collection first before; should this be consistent with that (and all the other times you listed collection first), or is this intentionally ordered in this way? If it isn't intentional, I recommend keeping consistent with the collection->exploration ordering. If it is intentional, I recommend adding an HTML comment briefly explaining the order is intentional and why.